### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by ToshY** [bpolaszek/meiliweb#3] [Ideas]
The demo looks really nice and with so much more functionality than the basic UI meilisearch ships with 👏 

I would love to try it out for my existing meilisearch instances, but it seems there currently isn't a docker image for it yet. Would you consider creating one?

**Comment by bpolaszek** [comment]
Hello @ToshY, thanks for your support! 🙏 
I'm personally not a Docker expert, but I don't think it would be hard to set up, I'll have a look.

Anyway, you can already test it against your instances (provided they're exposed through HTTPS) on https://meiliweb.pages.dev.
PR welcome otherwise! 🙂

**Reply by bpolaszek** [reply]
Just had a quick look - https://github.com/bpolaszek/meiliweb/pull/4 would this fit?

**Reply by ToshY** [reply]
I can try to take a look at it, but I honestly don't have much knowledge on Dockerfiles when it comes to dockerizing Javascript applications.
